### PR TITLE
Version 1.6.0

### PR DESCRIPTION
**Pull request by diddledan**
also uses gnome-platform - works on my system when built with cleanbuild

### CODE REVIEW

**Comment by Ads20000** [comment]
Snapcraft command on Ubuntu 17.04 failed. Snapcraft cleanbuild works though and the snap built from that works. HOWEVER, it does not run. Even after `snap install gnome-3-24` and `snap connect corebird:gnome-3-24-platform gnome-3-24:gnome-3-24-platform` (which is not user-friendly at all, someone who installs via their software center and launches via their launcher will not see his message, Corebird will simply not work, but that's some design work that needs to happen on snappy's side with the GNOME platform) it still says that I haven't connected this snap with the GNOME platform snap.
```
$ snapcraft
Skipping pull desktop-gnome-platform (already ran)
Skipping pull libopenh264-library (already ran)
Skipping build libopenh264-library (already ran)
Skipping pull gnome-platform (already ran)
'gstreamer' has prerequisites that need to be staged: libopenh264-library gnome-platform
Preparing to build gnome-platform 
Building gnome-platform 
tee: /etc/apt/sources.list.d/gnome-3-24.list: Permission denied
deb http://ppa.launchpad.net/ubuntu-desktop/gnome-3-24/ubuntu xenial main
Executing: /tmp/apt-key-gpghome.VBD8dfZvQO/gpg.1.sh --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 399B698EEA9EF163B6F9A0F62CC98497A1231595
gpg: key 2CC98497A1231595: public key "Launchpad PPA for Ubuntu Desktop" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no writable keyring found: Not found
gpg: error reading '[stdin]': General error
gpg: import from '[stdin]' failed: General error
gpg: Total number processed: 0
W: chmod 0700 of directory /var/lib/apt/lists/partial failed - SetupAPTPartialDirectory (1: Operation not permitted)
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
Command '['/bin/sh', '/tmp/tmpyuxvzf3g', '/tmp/tmpgnngvfs7']' returned non-zero exit status 100.
$ sudo snap install gnome-3-24
gnome-3-24 3.24 from 'canonical' installed
$ sudo snap connect corebird:gnome-3-24-platform gnome-3-24:gnome-3-24-platform
$ snap run corebird
You need to connect this snap to the gnome platform snap.

You can do this with those commands:
snap install gnome-3-24
snap connect corebird:gnome-3-24-platform gnome-3-24:gnome-3-24-platform
```

**Comment by diddledan** [comment]
The issue with reporting that the gnome-platform snap isn't connected even when you've clearly done the required command is a bug in snapd which 2.27 is supposed to fix when it's available. In the meantime you can run `sudo /usr/lib/snapd/snap-discard-ns corebird` though that is a low-level internal-to-snapd command that shouldn't be used ordinarily by users

**Comment by Ads20000** [comment]
Yes apologies I found the relevant bug online, let me try again.

**Comment by Ads20000** [comment]
Interesting, `SNAPCRAFT_CONTAINER_BUILDS=1 snapcraft` fails when trying to build `gstreamer`, however `snapcraft cleanbuild` works fine (haven't tested on Ubuntu 16.04 though, have you?). Do you have write access to this repo so you can pull it? The sooner we get this out to `stable` the better!

```
Parts 'libopenh264-library' and 'gstreamer' have the following file paths in common which have different contents:
    lib/x86_64-linux-gnu/libwrap.so.0.7.6
    usr/lib/x86_64-linux-gnu/libFLAC.so.8.3.0
    usr/lib/x86_64-linux-gnu/libasyncns.so.0.3.1
    usr/lib/x86_64-linux-gnu/libpulse-simple.so.0.1.0
    usr/lib/x86_64-linux-gnu/libvorbis.so.0.4.8
    usr/lib/x86_64-linux-gnu/libvorbisenc.so.2.0.11
    usr/share/doc/gcc-6-base/README.Debian.amd64.gz
    usr/share/doc/gcc-6-base/changelog.Debian.gz
    usr/share/doc/libasyncns0/changelog.Debian.gz
    usr/share/doc/libdbus-1-3/NEWS.gz
    usr/share/doc/libdbus-1-3/changelog.Debian.gz
    usr/share/doc/libflac8/changelog.Debian.gz
    usr/share/doc/libflac8/copyright
    usr/share/doc/libpulse0/NEWS.gz
    usr/share/doc/libpulse0/changelog.Debian.gz
    usr/share/doc/libpulse0/copyright
    usr/share/doc/libsndfile1/changelog.Debian.gz
    usr/share/doc/libsndfile1/copyright
    usr/share/doc/libvorbis0a/changelog.Debian.gz
    usr/share/doc/libwrap0/changelog.Debian.gz

Snapcraft offers some capabilities to solve this by use of the following keywords:
    - `filesets`
    - `stage`
    - `snap`
    - `organize`

Learn more about these part keywords by running `snapcraft help plugins`
```

**Comment by diddledan** [comment]
no, I don't have access to pull the PR. @flexiondotorg or @popey should be able to do so

**Comment by Ads20000** [comment]
Could I put a call for testing on the forum for 1.6.0 on Ubuntu 16.04? :)
